plugins/module_utils/foreman_helper.py

```diff
@@ -64,7 +64,7 @@
     hostgroups='title',
     locations='title',
     operatingsystems='title',
-    # TODO: Organizations should be search by title (as foreman allow nested orgs) but that's not the case ATM.
+    # TODO: Organizations should be search by title (as foreman allows nested orgs) but that's not the case ATM.
```

Comment:
Wondering, how much :cake: and :beers: it would take to remove that TODO.

Reply:
Last time my proposed expense was denied, something about "you don't need a country of the size of Luxembourg" :woman_shrugging: